src/Configuration/Loader.php

```diff
@@ -18,8 +20,12 @@ public function load(string $file): Configuration
             throw new MissingFileException();
         }
 
-        return Configuration::fromArray(
-            Yaml::parse(file_get_contents($file))
-        );
+        try {
+            $data = Yaml::parse(file_get_contents($file));
+        } catch (ParseException $exception) {
+            throw FileCannotBeParsedAsYamlException::fromFilename($file);
```

Comment:
An `@throws FileCannotBeParsedAsYamlException` annotation should be added to the method docBlock